assets/airports/cyow.json

```diff
@@ -0,0 +1,547 @@
+        "SEBEG":    ["N45d16m37.00", "W075d54m01.00"],
+        "MEGNO":    ["N45d21m51.00", "W075d56m83.00"],
+        "OMTUN":    ["N45d22m69.00", "W075d59m11.00"],
+        "ULAGU":    ["N45d11m31.00", "W075d43m66.00"],
+        "BOREG":    ["N45d14m57.00", "W075d36m43.00"],
+        "ADVOS":    ["N45d20m19.00", "W075d23m90.00"],
+        "KAREB":    ["N45d20m99.00", "W075d20m84.00"],
+        "DEMKO":    ["N45d10m45.00", "W075d39m97.00"],
+        "VOLAG":    ["N45d08m51.00", "W075d31m75.00"],
+        "DUKTI":    ["N45d07m32.00", "W075d29m48.00"],
+        "ART":      ["N43d57m13.00", "W076d03m88.00"],
+        "BUGSY":    ["N44d43m36.00", "W074d08m48.00"],
+        "SAVAL":    ["N44d54m25.00", "W074d42m02.00"],
+        "TAPVO":    ["N44d52m55.00", "W074d58m23.00"],
+        "CYRIL":    ["N44d57m17.00", "W075d24m15.00"],
+        "CAVUR":    ["N45d11m12.00", "W075d29m27.00"],
+        "LITRI":    ["N45d21m83.00", "W075d33m88.00"],
+        "APMUP":    ["N45d23m71.00", "W075d36m94.00"],
+        "MERKO":    ["N45d27m07.00", "W075d43m58.00"],
+        "ILOGI":    ["N45d29m63.00", "W075d48m39.00"],
+        "PETLU":    ["N45d30m81.00", "W075d50m69.00"],
+        "TEXEN":    ["N45d12m53.00", "W075d27m56.00"],
+        "AMAGI":    ["N45d14m20.00", "W075d37m27.00"],
+        "NOVUX":    ["N45d12m57.00", "W075d40m88.00"],
+        "TILOG":    ["N45d09m12.00", "W075d48m52.00"],
+        "EBNAP":    ["N45d07m91.00", "W075d50m77.00"],
+        "BEMOG":    ["N46d09m04.00", "W075d34m35.00"],
+        "ALIDO":    ["N45d44m43.00", "W075d40m77.00"],
+        "DUNUM":    ["N45d34m25.00", "W075d49m08.00"],
+        "RAFNO":    ["N45d25m55.00", "W075d52m64.00"],
+        "REVAL":    ["N45d27m69.00", "W075d39m57.00"],
+        "XOLTI":    ["N45d28m65.00", "W075d31m52.00"],
+        "IGTAD":    ["N45d29m73.00", "W075d29m12.00"],
+        "ILUBO":    ["N45d23m09.00", "W075d35m90.00"],
+        "WATTO":    ["N45d15m97.00", "W075d22m22.00"],
+        "TIDUL":    ["N45d15m41.00", "W075d21m02.00"],
+        "GABEV":    ["N45d28m80.00", "W075d52m80.00"],
+        "KEMLU":    ["N45d24m21.00", "W075d50m96.00"],
+        "SENNZ":    ["N45d20m57.00", "W075d49m50.00"],
+        "MODON":    ["N45d17m57.00", "W075d56m19.00"],
+        "VINRA":    ["N45d16m49.00", "W075d58m57.00"],
+        "SMARE":    ["N46d19m56.00", "W078d09m82.00"],
+        "TOXIP":    ["N45d55m76.00", "W077d04m07.00"],
+        "ONDOB":    ["N45d41m62.00", "W076d26m25.00"],
+        "YUL":      ["N45d36m94.00", "W073d58m26.00"],
+        "ALSET":    ["N45d36m62.00", "W074d30m00.00"],
+        "VILRO":    ["N45d36m95.00", "W072d42m93.00"],
+        "CATOG":    ["N45d55m03.00", "W072d53m03.00"],
+        "OBTAX":    ["N45d41m68.00", "W073d16m10.00"],
+        "THURO":    ["N45d33m42.00", "W074d54m47.00"],
+        "ONBAB":    ["N45d28m92.00", "W075d36m08.00"],
+        "TEFLY":    ["N45d24m43.00", "W075d27m71.00"],
+        "URLAG":    ["N45d21m55.00", "W075d20m89.00"]
```

Comment:
please sort these alphabetically unless there is a good reason they are ordered the way they are.